atala-prism-sdk/build.gradle.kts

```
@@ -17,23 +17,42 @@ plugins {
     id("org.jetbrains.kotlinx.kover") version "0.7.3"
 }
 
+kover {
+    useJacoco("0.8.11")
+    excludeJavaCode()
+    excludeInstrumentation {
+        packages("androidx.test.espresso", "androidx.test.ext")
+    }
+}
+
 koverReport {
+    filters {
+        excludes {
+            packages(
+                "io.iohk.atala.prism.protos",
+                "io.iohk.atala.prism.walletsdk.domain",
+                "ioiohkatalaprismwalletsdkpluto.data"
```

This should be like that
```suggestion
                "io.iohk.atala.prism.walletsdk.pluto.data"
```